source/loader/linux/lib_init.cpp

```diff
@@ -0,0 +1,22 @@
+/*
+ *
+ * Copyright (C) 2022 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ */
+
+#include "../ur_lib.h"
```

Comment:
Is there a way to avoid including relative paths?

Reply:
Done